spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -339,14 +339,13 @@ def main(args=None):
     if fname_disc:
         # Apply straightening to disc-label
         printv('\nApply straightening to disc labels...', verbose)
-        run_proc('isct_antsApplyTransforms -d 3 -i %s -r %s -t %s -o %s -n %s' %
+        run_proc('sct_apply_transfo -i %s -d %s -w %s -o %s -x %s' %
```

Comment:
same comment as in https://github.com/neuropoly/spinalcordtoolbox/pull/2990/files#r543841827, what is the reasoning for calling the script?

Reply:
ahhh i guess the reasoning is that this is a special use of `sct_apply_transfo` where you'd like to use the 'label' feature, right?

Reply:
That is the re reason yes !